sdk/nodejs/cmd/run-policy-pack/run.ts

```
@@ -201,7 +201,10 @@ export function run(opts: RunOpts): Promise<Record<string, any> | undefined> | P
 
         // Default message should be to include the full stack (which includes the message), or
         // fallback to just the message if we can't get the stack.
-        const defaultMessage = err.stack || err.message;
+        //
+        // If both the stack and message are empty, then just stringify the err object itself. This
+        // is also necessary as users can throw arbitrary things in JS (including non-Errors).
+        const defaultMessage = err.stack || err.message || ("" + err);
```

note: i'm not calling `.toString()` as you can throw `null` and `undefined`.